src/MOI_wrapper.jl

```diff
@@ -22,7 +25,7 @@ MOI.is_empty(optimizer::Optimizer) = isnothing(optimizer.solver) && isnothing(op
 function MOI.empty!(optimizer::Optimizer)
     optimizer.solver = nothing
     optimizer.nlp = nothing
-    optimizer.stats = nothing
+    # TODO how to I reset `stats` ?
```

Comment:
There is a SolverCore.reset! function for that : 
```suggestion
    reset!(stats)
```

Reply:
https://github.com/JuliaSmoothOptimizers/SolverCore.jl/blob/e310d911ea8de145979584c9af8ada2d02e77c90/src/stats.jl#L178

Reply:
Ahaha! So true